.github/workflows/packaging.yml

```diff
@@ -101,3 +101,74 @@ jobs:
           jupyter nbextension list 2>&1 | grep -ie "voila/extension.*enabled" -
           voila --version
           voila --help
+
+  release:
+    # TODO
+    # if: startsWith(github.ref, 'refs/tags/')
```

Comment:
This might not be required if the job is moved to its own workflow and triggered with:

```yaml
on:
  push:
    tags:
      - '*'
```